MyBlog.Services/Areas/Users/IUserService.cs

```diff
@@ -0,0 +1,16 @@
+﻿using MyBlog.Infrastructure.Entities;
+
+namespace MyBlog.Service.Areas.Users;
+
+public interface IUserService
+{
+    Task<List<User>> GetAllAsync();
```

Comment:
It`s  better to rename method to GetListAsync (if it names GetList it`s easy to understand, that you response all users and easy to understand response type). Try to give short naming to the methods).